src/main/java/org/prebid/server/auction/ExchangeService.java

```diff
@@ -215,6 +215,15 @@ private static ExtRequestTargeting targeting(ExtBidRequest requestExt) {
         return currency != null ? currency.getRates() : null;
     }
 
+    /**
+     * Extracts usepbsrates flag from {@link ExtBidRequest}.
+     */
+    private static boolean usepbsrates(ExtBidRequest requestExt) {
+        final ExtRequestPrebid prebid = requestExt != null ? requestExt.getPrebid() : null;
+        final ExtRequestCurrency currency = prebid != null ? prebid.getCurrency() : null;
+        return currency != null ? currency.getUsepbsrates() : false;
```

Comment:
currency.getUsepbsrates() retruns Boolean not boolean, so it can be null. 
I'd prefer if we extract this variable and return null.